zoom/help.md

```diff
@@ -222,6 +219,41 @@ Example output:
 }
 ```
 
+### Tasks
+
+#### Retrieve User Activity Events
+
+This task is used to retrieve zoom user activity events.
+
+It allows configuring schedules based on:
```

Comment:
```suggestion
Supported schedule types for this task include:
```
Based on `icon-plugin` changes suggested by you to avoid regeneration issues in future.

Reply:
Regenerated with latest release of icon-plugin